.github/workflows/wheels-Windows-MacOS.yml

```
@@ -0,0 +1,55 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Wheels for Windows and MacOS
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, macOS-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8] 
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
```

Please could you add newlines between each of these dashed sections for readability, like the other yml files?
```suggestion

    - name: Set up Python ${{ matrix.python-version }}
```